### DISCUSSION TITLE
Migration Workflow

### DISCUSSION THREAD

**Discussion by iroddis** [SeaQL/sea-orm#914] [Q&A]
Hello again. I know that the migration workflow is:

1. write the migration
2. migrate up
3. generate entities from DB

I've just figured out `ActiveEnum`s, but I need them to support `Copy` (similar to #668 ). I can easily add the `Copy` trait myself to the `sea_orm_active_enums.rs` as was suggested, but any time I write more migrations I'll lose all the customization when I do step (3) above.

What's the recommended way around this? Should I just keep a patch file to apply after generating?

**Comment by billy1624** [comment]
Hey @iroddis, sorry for the delay!

There is a ongoing discussion on the issue. Please join the discussion at
- https://github.com/SeaQL/sea-orm/discussions/668
